src/nikita/Assignment19Char.java

```diff
@@ -0,0 +1,30 @@
+package nikita;
+
+public class Assignment19Char {
+
+	int count;
+	char charAt;
+	
+	void frequencyOfChar(String input) {
+		for(int i=0; i<input.length();i++) {
+			if(input.charAt(i)=='t') {
```

Comment:
use generic process, your code will only work for character 't', you have to pass any character as a parameter and your code will print frequency of it